recipes/folly/all/patches/0016-find-packages-2.patch

```diff
@@ -0,0 +1,88 @@
+diff --git a/CMake/folly-deps.cmake b/CMake/folly-deps.cmake
+index 34e834d56..08f7712f7 100644
+--- a/CMake/folly-deps.cmake
++++ b/CMake/folly-deps.cmake
+@@ -52,21 +52,21 @@ find_package(DoubleConversion MODULE REQUIRED)
+ list(APPEND FOLLY_LINK_LIBRARIES ${DOUBLE_CONVERSION_LIBRARY})
+ list(APPEND FOLLY_INCLUDE_DIRECTORIES ${DOUBLE_CONVERSION_INCLUDE_DIR})
+ 
+-find_package(Gflags MODULE)
+-set(FOLLY_HAVE_LIBGFLAGS ${LIBGFLAGS_FOUND})
+-if(LIBGFLAGS_FOUND)
+-  list(APPEND FOLLY_LINK_LIBRARIES ${LIBGFLAGS_LIBRARY})
+-  list(APPEND FOLLY_INCLUDE_DIRECTORIES ${LIBGFLAGS_INCLUDE_DIR})
+-  set(FOLLY_LIBGFLAGS_LIBRARY ${LIBGFLAGS_LIBRARY})
+-  set(FOLLY_LIBGFLAGS_INCLUDE ${LIBGFLAGS_INCLUDE_DIR})
++find_package(gflags MODULE)
++set(FOLLY_HAVE_LIBGFLAGS ${gflags_FOUND})
++if(GFLAGS_FOUND)
++  list(APPEND FOLLY_LINK_LIBRARIES ${gflags_LIBS})
++  list(APPEND FOLLY_INCLUDE_DIRECTORIES ${gflags_INCLUDE_DIR})
++  set(FOLLY_LIBGFLAGS_LIBRARY ${gflags_LIBS})
++  set(FOLLY_LIBGFLAGS_INCLUDE ${gflags_INCLUDE_DIR})
+ endif()
+ 
+-find_package(Glog MODULE)
+-set(FOLLY_HAVE_LIBGLOG ${GLOG_FOUND})
+-list(APPEND FOLLY_LINK_LIBRARIES ${GLOG_LIBRARY})
+-list(APPEND FOLLY_INCLUDE_DIRECTORIES ${GLOG_INCLUDE_DIR})
++find_package(glog MODULE)
++set(FOLLY_HAVE_LIBGLOG ${glog_FOUND})
++list(APPEND FOLLY_LINK_LIBRARIES ${glog_LIBRARIES})
++list(APPEND FOLLY_INCLUDE_DIRECTORIES ${glog_INCLUDE_DIR})
+ 
+-find_package(LibEvent MODULE REQUIRED)
++find_package(Libevent MODULE REQUIRED)
+ list(APPEND FOLLY_LINK_LIBRARIES ${LIBEVENT_LIB})
+ list(APPEND FOLLY_INCLUDE_DIRECTORIES ${LIBEVENT_INCLUDE_DIR})
+ 
+@@ -79,13 +79,13 @@ if (ZLIB_FOUND)
+ endif()
+ 
+ find_package(OpenSSL MODULE REQUIRED)
+-list(APPEND FOLLY_LINK_LIBRARIES ${OPENSSL_LIBRARIES})
+-list(APPEND FOLLY_INCLUDE_DIRECTORIES ${OPENSSL_INCLUDE_DIR})
+-list(APPEND CMAKE_REQUIRED_LIBRARIES ${OPENSSL_LIBRARIES})
+-list(APPEND CMAKE_REQUIRED_INCLUDES ${OPENSSL_INCLUDE_DIR})
++list(APPEND FOLLY_LINK_LIBRARIES ${OpenSSL_LIBRARIES})
++list(APPEND FOLLY_INCLUDE_DIRECTORIES ${OpenSSL_INCLUDE_DIR})
++list(APPEND CMAKE_REQUIRED_LIBRARIES ${OpenSSL_LIBRARIES})
++list(APPEND CMAKE_REQUIRED_INCLUDES ${OpenSSL_INCLUDE_DIR})
+ check_function_exists(ASN1_TIME_diff FOLLY_HAVE_OPENSSL_ASN1_TIME_DIFF)
+-list(REMOVE_ITEM CMAKE_REQUIRED_LIBRARIES ${OPENSSL_LIBRARIES})
+-list(REMOVE_ITEM CMAKE_REQUIRED_INCLUDES ${OPENSSL_INCLUDE_DIR})
++list(REMOVE_ITEM CMAKE_REQUIRED_LIBRARIES ${OpenSSL_LIBRARIES})
++list(REMOVE_ITEM CMAKE_REQUIRED_INCLUDES ${OpenSSL_INCLUDE_DIR})
```

Comment:
This should not be needed at all, because openssl recipe sets the correct values.